src/Core/Entities/Organization.cs

```diff
@@ -72,6 +72,11 @@ public class Organization : ITableObject<Guid>, ISubscriber, IStorable, IStorabl
     public int? MaxAutoscaleSeats { get; set; } = null;
     public DateTime? OwnersNotifiedOfAutoscaling { get; set; } = null;
     public OrganizationStatusType Status { get; set; }
+    public bool UsePasswordManager { get; set; } = true;
+    public int? SmSeats { get; set; }
+    public int? SmServiceAccounts { get; set; }
+    public int? MaxAutoscaleSmSeats { get; set; } = null;
+    public int? MaxAutoscaleSmServiceAccounts { get; set; } = null;
```

Comment:
This is more of a general discussion topic, so feel free to reply elsewhere if this is the wrong spot 😄 
These classes seem pretty big. At what point can we say they have too much responsibility and need to be broken up?

Reply:
It's not a problem for this particular class, because the `Entities` classes have a 1-to-1 mapping with the database tables, which (afaik) is required by our ORM [Dapper](https://github.com/DapperLib/Dapper). They're basically C# representations of the table. So these classes will necessarily grow as our database tables get wider (which itself isn't a problem because databases are concerned with data normalization and not object oriented concepts of responsibility).

But you're right that we should consider returning smaller objects from the database that represent the data needed for a particular domain. This `Organization` object is the equivalent of `SELECT * FROM [dbo].[Organizations]`, which is starting to become overkill from an OOP perspective, we could query just the columns we need for a given domain and return a smaller, more relevant object instead. I think it's out of scope of this PR but something we should consider.

Reply:
Thanks for the insight!